iFixit/react-commerce
# Use iFixit API for default device images

fixes #247 

## QA

1. Visit the Vercel PR preview (replace `vercel.app` with `cominor.com`)
2. Visit the parts product list
3. Verify that `/Store/Parts/Kindle_Tablet` has an hero image, even though the Strapi product list for `Kindle_Tablet` has no image set
